lang/en.json

```diff
@@ -86,6 +86,9 @@
   "TAB.NOTE": "NOTE",
   "TAB.BUILDING": "BUILDING",
   "TAB.HIRELING": "HIRELING",
+  "TAB.ATTRIBUTES": "ATTRIBUTES",
+  "TAB.SUPPLY": "SUPPLY",
```

Comment:
Well, what you have in this tab covers as much crafting as it is about purchasing.
I liked the sound of "supply" because − I think − it sounds generic enough to cover both. What do you think?

Reply:
I think a name that isn't also the name of a field is better (unless the other fields on the tab are related to that field -- which I don't really think they are), but my two cents are as much worth as yours here. What about something like "ACQUISITION" then, if the tab is about how one would acquire this item? But you should definitely go with "SUPPLY" if that sounds best to you.

Reply:
Funny enough: I went for « _acquisition_ » in the french version (we don't really have a nice and direct equivalent for « supply ») with 4148defd
I'll make the change I think.

While we're at it, if you have something something swedish for both ~~`TAB.SUPPLY`~~ `TAB.ACQUISITION` and `TAB.ARTIFACT`, I would be much obliged. 😄🇸🇪

Reply:
In my opinion SUPPLY, PROVISION, or ACQUISITION. The thing about Supply is that it is already a term in game, and so to avoid conflict/confusion the other two work better. ACQUISITION is in my opinion rather unwieldy. The tab title should in my opinion be called "Craft & Supply" if it can fit.